week-15/quizzes/05-thunk-quiz.md

```diff
@@ -0,0 +1,93 @@
+# Thunk Quiz
+
+This quiz is meant to check your understanding of thunks.
+
+## The questions
+
+<quiz>
+  <question>
+    <p>What is a thunk?</p>
+    <answer correct>A thunk is a function that gets returned by another
+    function, the thunk action creator.</answer>
```

Comment:
"..., also known as the thunk action creator." I added those words to make it flow better.